packages/synapse-react-client/src/components/SynapseTable/TopLevelControls/TopLevelControlsUtils.ts

```diff
@@ -31,25 +33,24 @@ export function getNumberOfResultsToInvokeAction(
  *   - If the total number of query results is unknown, the count is not included. e.g. '<unit>s'
  * If the user has selected rows, return the number of selected rows. e.g. '2 <unit>s'
  * @param hasResettableFilters
- * @param isRowSelectionVisible
+ * @param hasSelectedRows
  * @param selectedRows
  * @param data
  * @param unitDescription
  */
 export function getNumberOfResultsToInvokeActionCopy(
   hasResettableFilters: QueryContextType['hasResettableFilters'],
-  isRowSelectionVisible: QueryVisualizationContextType['isRowSelectionVisible'],
+  hasSelectedRows: QueryVisualizationContextType['isRowSelectionVisible'],
```

Comment:
Supposed to be QueryVisualizationContextType['hasSelectedRows']?

Reply:
good catch